### ISSUE TITLE
Seperate values when syncing to native SharpSpring endpoints

### ISSUE THREAD

**Issue by strandofgenius** [verbb/formie#1073]
### What are you trying to do?

When using the built-in fields such as the address and name (when using the multi-field option) it is not possible to map each value to a single field in SharpSpring. 

For example, the output below is a single address field value which is not going to let me select individual SharpSpring fields to set a city, state, zip etc.

`{"autocomplete":null,"address1":"323 Bauch Path\nLake Carson, RI 00193-9716","address2":"960","address3":"Alley","city":"Lake Alvena","state":"Minnesota","zip":"48650-9235","country":"Gibraltar"}`

### What's your proposed solution?

I'd like to see if there is a way to have the output be individual field names to map them in SharpSpring.

**Comment by engram-design** [comment]
Are you mapping to a contact, or a form?

For a contact, you can pick whatever field you like to populate the individual components of the SharpSpring address field. Here's an example where I'm using a Formie address field.

![image](https://user-images.githubusercontent.com/1221575/188241668-62856a46-9e08-4510-9974-d01c0d5bf096.png)

If we're talking about mapping fields for a Form, that should also be working:

![image](https://user-images.githubusercontent.com/1221575/188242357-d0eceb6b-c92f-4370-9fc4-b5bfc8d95008.png)

**Comment by strandofgenius** [comment]
I am mapping to a form. From the CP I enabled the integration, added my endpoint. And clicked sync which does send test data to the form endpoint, but it is not broken out like your screenshot. The address field and name field are just as I pasted in my initial comment. Other fields included like email, phone, checkboxes, drop downs work as expected.

**Comment by engram-design** [comment]
Can you also provide what version of Formie you're on? That has been submitted as a feature request issue, but would be good to know more about your setup.

I can see the issue now, to do with the special serialization we do for forms. Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by strandofgenius** [comment]
The first time I tried it for this new project was on version 2.0.0-beta.17 and then again on 2.0.7 on a Craft 4 install.

I apologize for the mix up on requesting for a feature vs asking it already existed.

**Comment by engram-design** [comment]
Fixed in [2.0.10](https://github.com/verbb/formie/releases/tag/2.0.10)

**Comment by strandofgenius** [comment]
@engram-design this is still an issue on Craft 3.
